### PR TITLE
Game logic

### PR DESCRIPTION
**Pull request by marilirulita**
Add real logic for tic-tac-toe game
Building with methods block

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
# **STATUS: CHANGES REQUIRED**

Hey **@marilirulita**  and **@diamond-nicholas**, I applaud you for the splendid job done so far. However, there are a few changes to make to get the milestone approved.

### Required Changes:

- [x] The project should use at least two classes. You can have Game and Player classes. Add these to the `lib` folder. Remember to use ` attr_readers/attr_writers` in the classes. All the game logic should be within the classes.
- [ ] Apparently, the game logic is missing the critical concept of determining a draw or a win. So far, every game is a draw. Kindly add that logic bit to make your game even smarter.
- [x] Kindly delete all inline commented code in your file. Remember to have none in the `lib` folder once you create the classes.

**Make the above changes and submit another CRR.** 

**Cheers and Happy coding!**

Reach me on slack:
**Roy Mukuye**

File: bin/main.rb [hunk 1]
```diff
@@ -2,53 +2,145 @@
 
 # class Tictactoe
```
Comment (mke2111) [inline bin/main.rb]: Kindly delete all commented ruby code.

**Review by BrianSammit — CHANGES_REQUESTED** [review]
CHANGE REQUIRED :recycle: 
---------------
Hello team great job so far!!! :clap: 
However, there is one change to fully complete this milestone :hammer_and_wrench: 
--------------------------

- [x] The game is telling who is the winner after the move from the player that lose. 

Screenshot from 2021-02-04 14-24-00

As you can see on the image the 'x' player won the game, but the game tells that after the next move. 

----------------
Kindly fix this issue and send it for another review :muscle: 
HAPPY CODDING!!!

**Review by GzimAsani — APPROVED** [review]
> # **STATUS: APPROVED** 
> Congratulations **@marilirulita **  and **@diamond-nicholas** diamond-nicholas , Great job implementing all the changes asked from the reviewers.

- [ OPTIONAL] It would be great if your game would ask for players names


<hr>
### if you have any question kindly reach me on slack @GzimAsani

### HAPPY CODING :computer:

**Review by GzimAsani — COMMENTED** [review]
# [STATUS] CHANGES REQUIRED♻️
Hello **ybadmus**, great job implementing all the changes requested from the previous reviewers but in order for you project to get accepted please fix the following:
- [ ] Your game never displays a winner or draw, again as mentioned before the game logic is not necessary so at some point in the game you can just display  one of the players to be declared a winner. In the current state the game does not display any winner even after all the moves were used.

### **Please make the required changes and submit another review request when you are ready.**

<hr>
### if you have any question kindly reach me on slack @GzimAsani

### HAPPY CODING :computer:

**Pull request merged**